hip_data_tools/aws/athena.py

```diff
@@ -110,12 +110,16 @@ def _show_result(self, execution_id, max_result_size=1000):
             [[val['VarCharValue'] for val in row['Data']] for row in results['ResultSet']['Rows']])
 
     def _get_query_result(self, execution_id, max_result_size=1000):
-        athena = self.conn.client(self.boto_type)
+        athena = self._client()
         results = athena.get_query_results(QueryExecutionId=execution_id,
                                            MaxResults=max_result_size)
         # TODO: Add ability to parse pages larger than 1000 rows
```

Comment:
are you keeping TODOs in line or at the top of the file?

Reply:
inline

Reply:
also, I don't touch them ever.